pyproject.toml

```diff
@@ -264,10 +249,16 @@ lint.select = [
     "NPY201", # numpy 2.0 compatibility (see https://numpy.org/devdocs/numpy_2_0_migration_guide.html#ruff-plugin) -- Explicitly listed to view preview rule errors
 ]
 lint.ignore = [
+    # formatting related ignores
+    # https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
+    "W191", # tab-identation
+    "E111", # indentation-with-invalid-multiple
+    "E114", # indentation-with-invalid-multiple-comment
+    "E117", # over-idented
+    "E501", # line-too-long
```

Comment:
I see we have this rule turned off in black. It seems like the lines are much longer here though. What happens if we turn this on? Maybe there will be too many long strings to easily ignore? If rough can add ignores for us, I'd like to turn this on.

Reply:
```console
ruff --select E501 --statistics
13482   E501    Line too long (100 > 88)
```